pom.xml

```diff
@@ -0,0 +1,227 @@
+                                <filter>${basedir}/src/main/assembly/filter.properties</filter>
+                            </filters>
+                            <descriptors>
+                                <descriptor>src/main/assembly/zip-assembly.xml</descriptor>
+                            </descriptors>
+                            <appendAssemblyId>false</appendAssemblyId>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>5.1.8</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <_nouse.jdk.javac>true</_nouse.jdk.javac>
+                        <Import-Package>
+                            io.grpc.*,
+                            com.google.protobuf.*,
+                            org.wso2.carbon.*,
+                            org.apache.synapse.*,
+                            *
+                        </Import-Package>
+                        <Embed-Dependency>
+                            perfmark-api;scope=compile|runtime,
+                            protobuf-java;scope=compile|runtime,
+                            grpc-protobuf;scope=compile|runtime,
+                            grpc-stub;scope=compile|runtime,
+                            grpc-api;scope=compile|runtime,
+                            grpc-core;scope=compile|runtime,
+                            grpc-netty-shaded;scope=compile|runtime
+                        </Embed-Dependency>
+                        <Export-Package>
+                            !com.google.protobuf.*,
+                            io.grpc.*;-split-package:=merge-first
+                        </Export-Package>
+                        <Include-Resource>{maven-resources},META-INF/services=target/classes/META-INF/services</Include-Resource>
+                        <_includeresource>META-INF/services=target/classes/META-INF/services</_includeresource>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <repositories>
+        <repository>
+            <id>wso2-nexus</id>
+            <name>WSO2 Nexus Repository</name>
+            <url>https://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>ignore</checksumPolicy>
+            </releases>
+        </repository>
+    </repositories>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2-releases</id>
+            <name>WSO2 Release Repository</name>
+            <url>https://maven.wso2.org/nexus/service/local/staging/deploy/maven2/</url>
```

Comment:
Lets use this url. https://github.com/wso2-extensions/esb-inbound-kafka/blob/master/pom.xml#L195